GitUI/UserControls/BlameControl.cs

```diff
@@ -388,6 +434,48 @@ private void BuildAuthorLine(GitBlameLine line, StringBuilder lineBuilder, strin
             }
         }
 
+        public const int BlameGradientCount = 20;
+        public static IEnumerable<Color> GetGradients(Color start, Color end, int steps = BlameGradientCount)
+        {
+            var stepA = (end.A - start.A) / (steps - 1);
+            var stepR = (end.R - start.R) / (steps - 1);
+            var stepG = (end.G - start.G) / (steps - 1);
+            var stepB = (end.B - start.B) / (steps - 1);
+
+            for (int i = 0; i < steps; i++)
+            {
+                yield return Color.FromArgb(start.A + (stepA * i),
+                    start.R + (stepR * i),
+                    start.G + (stepG * i),
+                    start.B + (stepB * i));
```

Comment:
.AdaptBackColor()

Reply:
Sorry, I have pushed a change at this code nearly at the same time of your comment.
Is this change (that I don't fully understand) really needed now that we no more write text on it?

Reply:
The colors still look a little strange even if just a small band, so I believe so.

Reply:
The 2 seems strange :(

Real heatmap colors we are used to (without `.AdaptBackColor()`):
![image](https://user-images.githubusercontent.com/460196/78307598-8ce1f300-7546-11ea-8e20-d617008cff35.png)

Adapted heatmap colors that no more looks like a heatmap (with `.AdaptBackColor()`):
![image](https://user-images.githubusercontent.com/460196/78308274-5dcc8100-7548-11ea-9cb2-968188a7e167.png)

Surely that I must use other colors but I really don't know which...

Reply:
Perhaps better:
![image](https://user-images.githubusercontent.com/460196/78309070-5ad29000-754a-11ea-8273-4446ca59d620.png)

Reply:
overall liking the direction, though I'd go for the green color set
![Screenshot_20200403-172152](https://user-images.githubusercontent.com/4403806/78331428-86b73b00-7575-11ea-982a-cc2f136a77f8.png)

Reply:
dark green being the most recent, and whitish the oldest

Reply:
for the dark version I think we'd need to invert, light green - the youngest, dark grey the oldest

Reply:
> for the dark version I think we'd need to invert, light green - the youngest, dark grey the oldest

That is basically what the .AdaptBackColor() does

It is easier to handle theming with a minimal of specific changes
Green with Adapt looks OK for both normal/dark theme

green without adapt

![image](https://user-images.githubusercontent.com/6248932/78335163-62ffef00-758d-11ea-941d-a33f7b2e32cd.png)

![image](https://user-images.githubusercontent.com/6248932/78334681-7d859880-758c-11ea-8670-b50c76f2e850.png)

Adapt

![image](https://user-images.githubusercontent.com/6248932/78334848-c5a4bb00-758c-11ea-9139-9c11f6d7239b.png)

Reply:
Question: How to easily get if the current theme is mostly dark or light to adapt the colors?

Reply:
`ColorHelper.IsLightTheme()`

Reply:
Use ``ColorHelper.AdaptColor(c, isForeground);`` whenever possible so that the color is chosen automatically. The advantage is, the result changes if the theme changes.